packaging/debian/build_debian.sh

```diff
@@ -52,18 +69,31 @@ build_one_architecture() {
     > "$dest/build/DEBIAN/control"
   cp "$here/postinst" "$dest/build/DEBIAN/"
 
+  # Now the Electron build.
+  echo "Building Electron client for $electron_arch"
+  backto=`pwd`
+  cd ../../desktop
+  node package.js --platform linux --arch $electron_arch
+  cp -r release/linux-${electron_arch}/Keybase-linux-${electron_arch}/* "$dest/build/opt/keybase"
```

Comment:
Is there any chance that one of the build artifacts here will ever have a space in it? If so, I don't know a good way to quote wildcards in bash, but we have the option of using some `xargs` voodoo or switching to `rsync` :(